node_modules/.bin/autoprefixer.ps1

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env pwsh
+$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent
+
+$exe=""
+if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
+  # Fix case when both the Windows and Linux builds of Node
+  # are installed in the same directory
+  $exe=".exe"
+}
```

Comment:
you had pushed node_modules then you added it to `.gitignore` Always try to add the  ignored files or folder to the `.gitignore` file before you push, so try to remove the node_modules from the repo it makes it unprofessional and confusing with a lot of files

Reply:
Sure i will remember that and i will also remove node_module from the repo